scripts/oncall-email/oncall.ml

```diff
@@ -0,0 +1,193 @@
+open Common
```

Comment:
Please include a summary of what this file is about, like we all should do for all source files.